tests/alarms.rs

```diff
@@ -486,94 +556,140 @@ mod alarm1_day {
         h24,
         set_alarm1_day,
+        ALARM1_SECONDS,
+        [4, 3, 2, 1],
         DayAlarm1 {
             day: 1,
             hour: Hours::H24(2),
             minute: 3,
             second: 4
         },
-        A1M::AllMatch,
-        ALARM1_SECONDS,
-        [4, 3, 2, 1]
+        A1M::AllMatch
     );
     set_alarm_test!(
         am,
         set_alarm1_day,
+        ALARM1_SECONDS,
+        [4, 3, 0b0100_0010, 1],
         DayAlarm1 {
             day: 1,
             hour: Hours::AM(2),
             minute: 3,
             second: 4
         },
-        A1M::AllMatch,
-        ALARM1_SECONDS,
-        [4, 3, 0b0100_0010, 1]
+        A1M::AllMatch
     );
     set_alarm_test!(
         pm,
         set_alarm1_day,
+        ALARM1_SECONDS,
+        [4, 3, 0b0110_0010, 1],
         DayAlarm1 {
             day: 1,
             hour: Hours::PM(2),
             minute: 3,
             second: 4
         },
-        A1M::AllMatch,
+        A1M::AllMatch
+    );
+    set_alarm_test!(
+        match_hms_naivetime,
+        set_alarm1_hms,
         ALARM1_SECONDS,
-        [4, 3, 0b0110_0010, 1]
+        [4, 3, 2, AM | 1],
+        NaiveTime::from_hms(2, 3, 4)
     );
-
     set_alarm_test!(
         match_hms,
         set_alarm1_day,
+        ALARM1_SECONDS,
+        [4, 3, 2, AM | 1],
         DayAlarm1 {
             day: 1,
             hour: Hours::H24(2),
             minute: 3,
             second: 4
         },
-        A1M::HoursMinutesAndSecondsMatch,
+        A1M::HoursMinutesAndSecondsMatch
+    );
+    set_alarm_test!(
+        match_hms_ign_incorrect_day,
+        set_alarm1_day,
         ALARM1_SECONDS,
-        [4, 3, 2, AM | 1]
+        [4, 3, 2, AM | 1],
+        DayAlarm1 {
+            day: 0,
+            hour: Hours::H24(2),
+            minute: 3,
+            second: 4
+        },
+        A1M::HoursMinutesAndSecondsMatch
     );
+
     set_alarm_test!(
         match_ms,
         set_alarm1_day,
+        ALARM1_SECONDS,
+        [4, 3, AM | 2, AM | 1],
         DayAlarm1 {
             day: 1,
             hour: Hours::H24(2),
             minute: 3,
             second: 4
         },
-        A1M::MinutesAndSecondsMatch,
+        A1M::MinutesAndSecondsMatch
+    );
+    set_alarm_test!(
+        match_ms_ign_incorrect_day,
+        set_alarm1_day,
         ALARM1_SECONDS,
-        [4, 3, AM | 2, AM | 1]
+        [4, 3, AM | 2, AM | 1],
+        DayAlarm1 {
+            day: 0,
+            hour: Hours::H24(2),
+            minute: 3,
+            second: 4
+        },
+        A1M::MinutesAndSecondsMatch
     );
     set_alarm_test!(
         match_s,
         set_alarm1_day,
+        ALARM1_SECONDS,
+        [4, AM | 3, AM | 2, AM | 1],
         DayAlarm1 {
             day: 1,
             hour: Hours::H24(2),
             minute: 3,
             second: 4
         },
-        A1M::SecondsMatch,
+        A1M::SecondsMatch
+    );
+    set_alarm_test!(
+        match_s_ign_incorrect_min,
```

Comment:
Could you please replace "`_ign_`" with "`_ignore_`" in the tests? That word seems to important to be missed.